core/jvm/src/main/scala/zio/sql/delete.scala

```diff
@@ -4,6 +4,8 @@ trait DeleteModule { self: ExprModule with TableModule =>
 
   sealed case class DeleteBuilder[F[_], A, B](table: Table.Aux[A]) {
     def where[F1](expr: Expr[F1, A, Boolean]): Delete[F1, A] = Delete(table, expr)
+
+    def all[F1]: Delete[Features.Literal, A] = Delete(table, Expr.literal(true))
```

Comment:
So, this isn't really nice from the usability perspective. What if we removed `DeletBuilder` entirely, moved the `where` method to the `Delete` (just like `Select` has) and made `deleteFrom` return `Delete` right away (with `where` equal true)?

Reply:
Let's do that.